server/channels/jobs/batch_migration_worker.go

```diff
@@ -111,7 +111,8 @@ func (worker *BatchMigrationWorker) IsEnabled(_ *model.Config) bool {
 // checkIsClusterInSync returns true if all nodes in the cluster are running the same version,
 // logging a warning on the first mismatch found.
 func (worker *BatchMigrationWorker) checkIsClusterInSync() bool {
```

Comment:
Should `checkIsClusterInSync` accept a `request.CTX` instead?

Reply:
it looks better !